Build/libHttpClient.Apple.C/libHttpClient.xcodeproj/project.pbxproj

```diff
@@ -1004,6 +1004,7 @@
 			hasScannedForEncodings = 0;
 			knownRegions = (
 				en,
+				Base,
```

Comment:
what does Base stand for?

Reply:
I'm not 100% sure, but there was a build warning for "enable base internationalization" that i added this to silence. It's ultimately irrelevant since libHttpClient (nor XAL) includes any strings, so all this does is silence a warning.